### ISSUE TITLE
Kali Linux 2020.3a 64 bit MHS35 Screen not working.

### ISSUE THREAD

**Issue by slimPickens** [lcdwiki/LCD-show-kali#9]
After initially installing and bricking my pi, I realized that the config.txt file was missing the 64bit flags and adding those back  solved the bricking problem but, now it locks up on the login screen and does not allow any input via usb.  

Anyone successfully install the MHS35 Screen with 64bit 2020.3a?

```

#For more options and information see
#http://rpf.io/configtxt
#Some settings may impact device functionality. See link above for details

#uncomment if you get no picture on HDMI for a default "safe" mode
#hdmi_safe=1

#uncomment this if your display has a black border of unused pixels visible
#and your display can output without overscan
#disable_overscan=1

#uncomment the following to adjust overscan. Use positive numbers if console
#goes off screen, and negative if there is too much border
#overscan_left=16
#overscan_right=16
#overscan_top=16
#overscan_bottom=16

#uncomment to force a console size. By default it will be display's size minus
#overscan.
#framebuffer_width=1280
#framebuffer_height=720

#uncomment if hdmi display is not detected and composite is being output
#hdmi_force_hotplug=1

#uncomment to force a specific HDMI mode (this will force VGA)
#hdmi_group=1
#hdmi_mode=1

#uncomment to force a HDMI mode rather than DVI. This can make audio work in
#DMT (computer monitor) modes
#hdmi_drive=2

#uncomment to increase signal to HDMI, if you have interference, blanking, or
#no display
#config_hdmi_boost=4

#uncomment for composite PAL
#sdtv_mode=2

#uncomment to overclock the arm. 700 MHz is the default.
#arm_freq=800

#Uncomment some or all of these to enable the optional hardware interfaces
#dtparam=i2c_arm=on
#dtparam=i2s=on
#dtparam=spi=on

#Uncomment this to enable infrared communication.
#dtoverlay=gpio-ir,gpio_pin=17
#dtoverlay=gpio-ir-tx,gpio_pin=18

#Additional overlays and parameters are documented /boot/overlays/README

#Enable audio (loads snd_bcm2835)
dtparam=audio=on

#USB mass storage boot is available on Raspberry Pi 2B v1.2, 3A+, 3B, and 3B+ only.
#program_usb_boot_mode=1

[pi2]
#Pi2 is 64bit only on v1.2+
#64bit kernel for Raspberry Pi 2 is called kernel8 (armv8a)
kernel=kernel8-alt.img
[pi3]
#64bit kernel for Raspberry Pi 3 is called kernel8 (armv8a)
kernel=kernel8-alt.img
[pi4]
#Enable DRM VC4 V3D driver on top of the dispmanx display stack
#dtoverlay=vc4-fkms-v3d
#max_framebuffers=2
#64bit kernel for Raspberry Pi 4 is called kernel8l (armv8a)
kernel=kernel8l-alt.img
[all]
#dtoverlay=vc4-fkms-v3d
#Tell firmware to go 64bit mode.
arm_64bit=1

hdmi_force_hotplug=1
dtparam=i2c_arm=on
dtparam=spi=on
enable_uart=1
dtoverlay=mhs35:rotate=90
hdmi_group=2
hdmi_mode=1
hdmi_mode=87
hdmi_cvt 480 320 60 6 0 0 0
hdmi_drive=2
```

**Comment by nexus166** [comment]
same here, no boot after running the script..

**Comment by slimPickens** [comment]
Finally got this working after following:

https://github.com/lcdwiki/LCD-show-kali/issues/6#issuecomment-711024983

so I'm closing this issue.

**Comment by kristiyandobrev** [comment]
Still getting the issue despite applying [this solution](https://github.com/lcdwiki/LCD-show-kali/issues/6#issuecomment-711024983)

I am also getting stucked on the Kali's login screen and as soon as I press any key on the usb attached mouse and keyboard the screen freezes. 

I am running `kali-linux-2021.1-rpi4-nexmon-64` and here is my config file: 

```
# For more options and information see
# http://rpf.io/configtxt
# Some settings may impact device functionality. See link above for details

# uncomment if you get no picture on HDMI for a default "safe" mode
#hdmi_safe=1

# uncomment this if your display has a black border of unused pixels visible
# and your display can output without overscan
#disable_overscan=1

# uncomment the following to adjust overscan. Use positive numbers if console
# goes off screen, and negative if there is too much border
#overscan_left=16
#overscan_right=16
#overscan_top=16
#overscan_bottom=16

# uncomment to force a console size. By default it will be display's size minus
# overscan.
#framebuffer_width=1280
#framebuffer_height=720

# uncomment if hdmi display is not detected and composite is being output
#hdmi_force_hotplug=1

# uncomment to force a specific HDMI mode (this will force VGA)
#hdmi_group=1
#hdmi_mode=1

# uncomment to force a HDMI mode rather than DVI. This can make audio work in
# DMT (computer monitor) modes
#hdmi_drive=2

# uncomment to increase signal to HDMI, if you have interference, blanking, or
# no display
#config_hdmi_boost=4

# uncomment for composite PAL
#sdtv_mode=2

#uncomment to overclock the arm. 700 MHz is the default.
#arm_freq=800

# Uncomment some or all of these to enable the optional hardware interfaces
#dtparam=i2c_arm=on
#dtparam=i2s=on
#dtparam=spi=on

# Uncomment this to enable infrared communication.
#dtoverlay=gpio-ir,gpio_pin=17
#dtoverlay=gpio-ir-tx,gpio_pin=18

# Additional overlays and parameters are documented /boot/overlays/README

# Enable audio (loads snd_bcm2835)
dtparam=audio=on

# USB mass storage boot is available on Raspberry Pi 2B v1.2, 3A+, 3B, and 3B+ only.
#program_usb_boot_mode=1

[pi2]
# Pi2 is 64bit only on v1.2+
# 64bit kernel for Raspberry Pi 2 is called kernel8 (armv8a)
kernel=kernel8-alt.img
[pi3]
# 64bit kernel for Raspberry Pi 3 is called kernel8 (armv8a)
kernel=kernel8-alt.img
[pi4]
# Enable DRM VC4 V3D driver on top of the dispmanx display stack
dtoverlay=vc4-fkms-v3d
max_framebuffers=2
# 64bit kernel for Raspberry Pi 4 is called kernel8l (armv8a)
kernel=kernel8l-alt.img
[all]
dtoverlay=vc4-fkms-v3d
# Tell firmware to go 64bit mode.
arm_64bit=1

# If you would like to enable USB booting on your Pi, uncomment the following line.
# Boot from microsd card with it, then reboot.
# Don't forget to comment this back out after using, especially if you plan to use
# sdcard with multiple machines!
# NOTE: This ONLY works with the Raspberry Pi 3+
#program_usb_boot_mode=1
hdmi_force_hotplug=1
dtparam=i2c_arm=on
dtparam=spi=on
enable_uart=1
dtoverlay=tft35a:rotate=90
```
